rj_gameplay/stp/utils/world_state_converter.py

```diff
@@ -265,10 +265,10 @@ def robot_creator(
     else:
         is_ours = True
         robot_id = robot_state.id
-        ball_sense = Optional[robot_status.has_ball_sense]
-        kicker_charged = Optional[robot_status.kicker_charged]
-        kicker_healthy = Optional[robot_status.kicker_healthy]
-        lethal_fault = Optional[robot_status.lethal_fault]
+        ball_sense = False
```

Comment:
actually I think this should be =robot_status.has_ball_sense, right? setting False always ignores the input when robot status is not None